Formula/vgmstream.rb

```diff
@@ -35,11 +49,15 @@ class Vgmstream < Formula
   fails_with gcc: "5" # ffmpeg is compiled with GCC
 
   def install
-    system "cmake", "-DBUILD_AUDACIOUS:BOOL=OFF", *std_cmake_args, "."
-    system "cmake", "--build", ".", "--config", "Release", "--target", "vgmstream_cli", "vgmstream123"
-    bin.install "cli/vgmstream-cli"
-    bin.install "cli/vgmstream123"
-    lib.install "src/libvgmstream.a"
+    ENV["LIBRARY_PATH"] = "#{HOMEBREW_PREFIX}/lib"
```

Comment:
```suggestion
    ENV["LIBRARY_PATH"] = HOMEBREW_PREFIX/"lib"
```
Why do we need to set this?

Reply:
vgmstream's [maintainer's advice](https://github.com/vgmstream/vgmstream/issues/1069#issuecomment-1030501736), I guess it's for compiler to find library installed from Homebrew. Without this variable, it won't compile.